app/app.go

```diff
@@ -220,11 +220,11 @@ func New(
 	govModuleAddr := authtypes.NewModuleAddress(govtypes.ModuleName).String()
 
 	app := &App{
-		BaseApp:       baseApp,
-		keys:          keys,
-		tkeys:         tkeys,
-		memKeys:       memKeys,
-		timeoutCommit: timeoutCommit,
+		BaseApp:   baseApp,
+		keys:      keys,
+		tkeys:     tkeys,
+		memKeys:   memKeys,
+		blockTime: timeoutCommit,
```

Comment:
Can we rename the timeoutCommit in the constructor as well (for consistency)

Reply:
see other comment, done in https://github.com/celestiaorg/celestia-app/pull/5639/commits/b8e2364290b10e071a2070063c6d870a63eb5bf9